File: sdkwork-backend-sdk-java/src/main/java/com/sdkwork/backend/model/PlusFileContentForm.java
package com.sdkwork.backend.model;

public class PlusFileContentForm {
    private Integer fileId;
    private String fileUuid;
    private String prompt;
    private String encoding;
    private FileContentObject content;

    public Integer getFileId() {
        return this.fileId;
    }
    
    public void setFileId(Integer fileId) {
        this.fileId = fileId;
    }

    public String getFileUuid() {
        return this.fileUuid;
    }
    
    public void setFileUuid(String fileUuid) {
        this.fileUuid = fileUuid;
    }

    public String getPrompt() {
        return this.prompt;
    }
    
    public void setPrompt(String prompt) {
        this.prompt = prompt;
    }

    public String getEncoding() {
        return this.encoding;
    }
    
    public void setEncoding(String encoding) {
        this.encoding = encoding;
    }

    public FileContentObject getContent() {
        return this.content;
    }
    
    public void setContent(FileContentObject content) {
        this.content = content;
    }
}

File: sdkwork-backend-sdk-csharp/Models/PlusFileContentForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusFileContentForm
    {
        public int? FileId { get; set; }
        public string? FileUuid { get; set; }
        public string? Prompt { get; set; }
        public string? Encoding { get; set; }
        public FileContentObject? Content { get; set; }
    }
}
